SwiftMonkeyPaws/Sources/Extensions/MonkeyPaws+Configuration.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  MonkeyPaws+Configuration.swift
+//  SwiftMonkeyPaws
+//
+//  Created by Daniel.Metzing on 11.02.18.
+//
+
+import UIKit
+
+extension MonkeyPaws {
+
+    public struct Configuration {
+        // Customise the appearance of the paws
+        public struct Paws {
+
+            /// Define the colour of the Paws
+            ///
+            /// - randomized: random colour for each paw
+            /// - constant: same colour for the paws
+            public enum Colour {
```

Comment:
I know this is me being picky but I would stick to the American English and just use *color* everywhere. This will avoid confusion especially that we already have `UIColor` and `CGColor` defined.